internal/cmd/stack/flags.go

```diff
@@ -99,3 +99,10 @@ var flagSearchCount = &cli.IntFlag{
 	Usage: "[Optional] Indicate the maximum count of elements returned when searching stacks",
 	Value: 30,
 }
+
+var flagVerbose = &cli.BoolFlag{
+	Name:     "verbose",
```

Comment:
I wonder if we'd be better just calling it `show-labels` for now given that's the only additional field we're adding? That ways it's really clear what it does, and doesn't get mistaken for something that sets the log level or something like that (on the grounds quite a few commands have a `verbose` option to increase the logging).

I suspect we're unlikely to want to include tons of extra columns in the table output because it very quickly becomes unreadable.